target/linux/mediatek/filogic/base-files/etc/board.d/02_network

```diff
@@ -55,6 +55,7 @@ mediatek_setup_interfaces()
 	dlink,aquila-pro-ai-m30-a1)
 		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4" internet
 		;;
+	cudy,m3000-v1|\
```

Comment:
Please add this in alphabetical order.

Reply:
I swear that dlink wasn't there when I rebased :grin: